source/dbo/Stored Procedures/Data Migration/uspMigrationOutcomeMismatch.sql

```diff
@@ -0,0 +1,38 @@
+﻿/*
+	Note that we treat some outcomes as being equal even when they are actually different in NTBS and ETS.
+	For more details see uspMigrationMisMatchedOutcomes.sql.
+*/
+
+CREATE PROCEDURE [dbo].[uspMigrationOutcomeMismatch]
```

Comment:
There is a similar sproc that already exists (`uspMigrationMisMatchedOutcomes.sql`) but that one's being used to aggregate the number of mismatched outcomes, whereas this one is showing more detail. I don't know if we'll still want the old one once we've updated the report?

Reply:
We can do the aggregation in SSRS from the detailed data so we shouldn't need both.

Reply:
This is now done